src/index.js

```diff
@@ -37,20 +39,24 @@ export default class ReactPasswordStrength extends React.Component {
     const { changeCallback, minScore, userInputs } = this.props;
     const password = this.refs['ReactPasswordStrength-input'].value;
 
-    let score;
+    let score, feedback;
 
     // always sets a zero score when min length requirement is not met
     // which avoids unnecessary zxcvbn computations (they require quite lots of CPU)
     if (this.isTooShort(password)) {
       score = 0;
+      feedback = {};
     } else {
-      score = zxcvbn(password, userInputs).score;
+      const result = zxcvbn(password, userInputs);
+      score = result.score;
+      feedback = result.feedback;
```

Comment:
I think what we'd prefer is to calculate the result and save it to a variable that we then pass to the `changeCallback` function. I expect ln 62 to be something like `changeCallback(this.state, result);`